examples/assets/BUILD.bazel

```diff
@@ -70,6 +80,7 @@ build_test(
         ":ts-out",
         ":out/src/index.js",
         ":out/src/styles.css",
+        ":out/src/generated.json",
```

Comment:
I think we do want the `:src/generated.json` up above in the "no root/out_dir"? I guess it would really be testing the `genrule` more then your change, but it makes the different test cases more consistent which I'd say is worth it...

Reply:
Same with the `root_dir` below?